.gitignore

```diff
@@ -30,6 +30,7 @@ loop
 !cmd/loop/
 
 loopd
+!loopd/
```

Comment:
We could possibly remove all three lines as there currently is no binary created in any folder inside the project (because there is no build command in the Makefile).

Reply:
Yep! https://github.com/lightninglabs/loop/pull/115/commits/a1a72507388aad812f26c40f7b8dd8384921f106